app/redux/smesher/actions.ts

```diff
@@ -70,9 +72,16 @@ export const deletePosData = () => async (dispatch: AppThDispatch) => {
   dispatch({ type: DELETED_POS_DATA });
 };
 
-export const pauseSmeshing = () => async (dispatch: AppThDispatch) => {
-  await eventsService.stopSmeshing({ deleteFiles: false });
-  dispatch({ type: PAUSED_SMESHING });
+export const pauseSmeshing = () => async (
+  dispatch: AppThDispatch,
+  getState: GetState
+) => {
+  const state = getState();
+  const postSetupState = getPostSetupState(state);
+  if (postSetupState === PostSetupState.STATE_IN_PROGRESS) {
```

Comment:
Here Im fixing a case when smeshing handlers for STOP smeshing are not registered yet, because has different from STATE_IN_PROGRESS status (error/stop/ finished etc.)
and we have SMESHER_STOP_SMESHING error.

Reply:
But it will make that User see the button, click on it, and it does nothing.

No-no, if the User clicks on the button and got an error — let him see this error. Then we will know that something is broken.
So probably you need to move this condition to the component, to not render the button when it is useless.

Reply:
added to the top level the error 


<img width="713" alt="Screenshot 2023-08-09 at 17 44 01" src="https://github.com/spacemeshos/smapp/assets/54288612/c3aaa289-11ba-43e8-9e63-bb8233b1c92e">

 I found out, that for resume smeshing we are handling the state and only with the right state we are restarting 
for pause smeshing, we are checking the state, only in 1 place, and showing this button. So, definitely, I have to handle it on my side too